### DISCUSSION TITLE
Blank page is displayed when trying to browse branch with '#' in branch name

### DISCUSSION THREAD

**Discussion by SnowSuno** [community/community#102761] [Repositories] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

A blank page is disabled when trying to view a specific branch with a `#` on its branch name.

> repro : https://github.com/sparcs-kaist/taxi-front/tree/%23189.2-image-zoom-pan-pinch

<img width="1424" alt="image" src="https://github.com/community/community/assets/77062919/752203af-ffda-4fac-aed4-3efcc733c140">

```
Uncaught TypeError: Cannot read properties of undefined (reading 'find')
    at oK (OverviewRepoFiles.tsx:29:32)
    at lb (react-dom.production.min.js:167:137)
    at od (react-dom.production.min.js:197:258)
    at i (react-dom.production.min.js:292:88)
    at uI (react-dom.production.min.js:280:389)
    at react-dom.production.min.js:280:320
    at uM (react-dom.production.min.js:280:325)
    at ux (react-dom.production.min.js:271:88)
    at uS (react-dom.production.min.js:268:429)
    at x (scheduler.production.min.js:13:203)
```

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Olayiwolaaa** [comment]
- The "#" symbol in your branch name might be causing issues. Some Git clients or web interfaces might not handle special characters in branch names well. Try renaming the branch to remove the "#" symbol and see if that resolves the issue.
- In rare cases, the branch itself might be corrupted. Try recreating the branch from scratch and pushing it to the remote repository.
- Try using a different interface or command-line Git to access the branch.
- **Check the Git logs:** Use the git log command to see if there are any errors or warnings associated with the #189.2-image-zoom-pan-pinch branch.
- Try viewing the branch locally

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @SnowSuno ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by SnowSuno** [comment, marked as answer]
The issue is solved, thank you!

**Reply by queenofcorgis** [reply]
@SnowSuno happy to hear, would you mind sharing the solution in case anyone else encounters this situation?

**Reply by SnowSuno** [reply]
I think that it was hotfixed by the github team.
